### ISSUE TITLE
BUG in initOptimScheduler.py

### ISSUE THREAD

**Issue by LiangzunFu** [OpenImagingLab/TimeLens-XL#7]
if params.init_step is not None:
            epoch = params.init_step if scheduler_type == 'epoch' else epoch / len_trainset
        for e in range(int(epoch) if scheduler_type == 'epoch' else int(epoch*len_trainset)):
            scheduler.step()
This will result in the learning rate decrease to the minimum at the beginning.
